<?xml version="1.0"?>

<launch>
    <node pkg="bumperbot_detection" type="get_coordinates.py" name="coordinate_detector" >
        <param name="launch_prefix" value="python3"/>
    </node>

    <node pkg="bumperbot_detection" type="coordinate_subscriber" name="coordinate_subscriber" output="screen" />

    <node pkg="bumperbot_detection" type="litter_coordinate_transformer_publisher_node" name="litter_coordinate_transformer_publisher_node" >
        <param name="base_frame"    value="base_footprint" />
        <param name="camera_frame"  value="dpcamera_link_optical"  />
    </node>

</launch>